app/models/activity.rb

```diff
@@ -7,6 +7,8 @@ class Activity < ActiveRecord::Base
 
   validates :content, :title, :team, presence: { if: ->(act) { act.kind == 'status_update' } }
 
+  delegate :students, to: :team
```

Comment:
No skipping is for pussies! I'll look into how to do this and will get back to you :+1: